### PR TITLE
Theme fix

### LINKED ISSUE #1866: Stats page display error
fireshot screen capture 148 - my community - statistics center - localhost_smf2_1_index_php_action stats

This is in Firefox ofc.

### PR DESCRIPTION
**Pull request by XinYenFon**
fixes #1866
Signed-off-by: Antes antes@simplemachines.org

### CODE REVIEW

File: Themes/default/css/admin.css [hunk 1]
```diff
@@ -57,24 +7,9 @@ body.action_admin .navigate_section ul li a {
 	border-top-left-radius: 0;
 	border-top-right-radius: 0;
 	box-shadow: none;
+	background: #F8F8F8;
```
Comment (live627) [inline Themes/default/css/admin.css:10]: I remove this for a reason...

Comment (XinYenFon) [inline Themes/default/css/admin.css:10]: which is?

Comment (live627) [inline Themes/default/css/admin.css:10]: 51acbe7607d866f2b843c1864ff0cccd038f91fc

Comment (XinYenFon) [inline Themes/default/css/admin.css:10]: I didn't ask when/where, i asked why :)

Comment (live627) [inline Themes/default/css/admin.css:10]: read the commit note

Comment (XinYenFon) [inline Themes/default/css/admin.css:10]: windowbg/2 using same bg color, so no point to add zebra-strips, also manage-boards has hover effect.

**Pull request merged**